python/hsfs/feature_view.py

```diff
@@ -3756,12 +3782,16 @@ def materialize_log(self, wait: Optional[bool] = False):
             materialization_result = feature_view.materialize_log(wait=True)
             ```
 
+        # Returns
+            Tuple(`Job`, `Job`) Job information for the materialization jobs of transformed/untransformed features
+
+
         # Raises
             `hsfs.client.exceptions.RestAPIError` in case the backend fails to materialize the log.
         """
-        return self._feature_view_engine.materialize_feature_logs(self, wait)
+        return self._feature_view_engine.materialize_feature_logs(self, wait, transformed)
 
-    def delete_log(self, transformed: Optional[bool] = None):
+    def delete_log(self, transformed: Optional[bool] = None) -> None:
```

Comment:
I don't think  `-> None` is needed, is it?

Reply:
why not?

Reply:
Isn't it the default?